superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -540,12 +540,14 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     fetchResource,
     createResource,
     updateResource,
+    clearError,
   } = useSingleViewResource<AlertObject>('report', t('report'), addDangerToast);
 
   // Functions
   const hide = () => {
     setIsHidden(true);
     onHide();
+    clearError();
```

Comment:
slight nit but let's do this before `onHide`, so we complete all internal logic before external logic